server/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs        = require('fs');
+const path      = require('path');
+const Sequelize = require('sequelize');
+const basename  = path.basename(module.filename);
+const env       = process.env.NODE_ENV || 'development';
+const config    = require(__dirname + '/../config/config.json')[env];
+const db        = {};
+
+let sequelize;
+if (config.use_env_constiable) {
+   sequelize = new Sequelize(process.env[config.use_env_constiable]);
+} else {
```

Comment:
The only thing i'm seeing in the if else is changed paramter to Sequelize class. can that be assigned to a variable instead?